packages/big-design/src/components/StatelessPagination/styled.tsx

```diff
@@ -0,0 +1,11 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { StyleableButton } from '../Button/private';
+
+export const StyledButton = styled(StyleableButton)`
```

Comment:
Since `defaultProps` is deprecated in React, we've been trying to migrate over to `.attrs({ theme: defaultTheme })` here instead.

Reply:
done: https://github.com/bigcommerce/big-design/pull/1598/commits/df1613855107fb6cff8e0a4014e8e206ece956c7